### PR TITLE
changes for developing applications section

### PR DESCRIPTION
**Pull request by joshuafernandes**
## Pull Request Description
Some new content and removed a fair amount of duplication re plugins; flow of docs now simplified and collated to what a user would do when they start to develop contracts, dapps etc

Flow is now like so: https://pegasys-docgoquorum--160.com.readthedocs.build/en/160/

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -110,8 +110,17 @@ nav:
       - Create a privacy-enabled network: Tutorials/Create-Privacy-enabled-network.md
       - Send private transactions: Tutorials/Send-private-transaction.md
       - Create a permissioned network: Tutorials/CreatePermissionedNetwork.md
-      - Connect to a node: Tutorials/Connecting-to-a-node.md
       - Use a plugin: Tutorials/Use-Plugin.md
+  - Developing Applications:
```
Comment (bgravenorst) [inline mkdocs.yml:114]: @joshuafernandes, would you be opposed to me moving this to How to > Develop applications?

Comment (bgravenorst) [reply mkdocs.yml:114]: As discussed, you based this on user feedback, so I'll keep this as is.

**Review by bgravenorst — APPROVED** [review]
LGTM. Made some editing changes.

**Comment by joshuafernandes** [comment]
Thankyou @bgravenorst :)

**Pull request merged**